### PR TITLE
ec2_instance: add missing metadata_options parameters

### PR DESCRIPTION
**Pull request by markuman**
##### SUMMARY
add missing metadata_options parameter
* http_put_response_hop_limit
* http_protocol_ipv6
* instance_metadata_tags

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
ec2_instance

### CODE REVIEW

**Comment by markuman** [comment]
> [Errno 28] No space left on device

cc @goneri

File: tests/integration/targets/ec2_instance/roles/ec2_instance/tasks/metadata_options.yml [hunk 1]
```diff
@@ -11,6 +12,35 @@
...
+      that:
+        - instance_creation is failed
+        - instance_creation is not changed
+        - "'This is required to set instance_metadata_tags' in instance_creation.msg"
+
+- name: test with boto3 version that supports instance_metadata_tags
+  vars:
+    ansible_python_interpreter: "{{ botocore_virtualenv_interpreter }}"
+  block:
+  - name: "fail create t3.nano instance with metadata_options"
```
Comment (alinabuzachis) [inline tests/integration/targets/ec2_instance/roles/ec2_instance/tasks/metadata_options.yml:31]: I guess the name of the task is misleading?

**Comment by markuman** [comment]
Still...

> : [Errno 28] No space left on device

**Comment by alinabuzachis** [comment]
recheck

**Comment by markuman** [comment]
recheck

**Review by alinabuzachis — APPROVED** [review]

**Pull request merged**